lib/Doctrine/Task/GenerateMigrationsDb.php

```diff
@@ -62,7 +62,7 @@ public function execute()
                 $connections[] = $connection->getName();
             }
             $changes = Doctrine_Core::generateMigrationsFromDiff($migrationsPath, $connections, $yamlSchemaPath);
-            $numChanges = count($changes, true) - count($changes);
+            $numChanges = count($changes, COUNT_RECURSIVE) - count($changes);
```

Comment:
The `COUNT_RECURSIVE` constant in PHP is equivalent to `1`

```
php > var_dump(COUNT_RECURSIVE);
int(1)
```
Which is what `true` would be coerced to in this case.

The other option is `COUNT_NORMAL` which evaluates to `0` (and is the default).

```
var_dump(COUNT_NORMAL);
int(0)
```